src/index.js

```diff
@@ -25,12 +25,12 @@ import { Fluence, KeyPair, setLogLevel } from '@fluencelabs/fluence';
 import { Elm } from './Main.elm';
 import * as serviceWorker from './serviceWorker';
 import { interfaceInfo, peerInfo } from './types';
-import { askAllAndSend } from './_aqua/app';
+import { askAllAndSend, getAll } from './_aqua/app';
 
-const defaultNetworkName = 'krasnodar';
+const defaultNetworkName = 'testNet + krasnodar';
 
 const defaultEnv = {
-    relays: [...krasnodar, ...testNet, ...stage],
+    relays: [...testNet, ...krasnodar, ...stage],
```

Comment:
Do we really want to expose our test nets here? Everything will break in we introduce an incompatible change to stage for example

Reply:
So, there are two kinds of incompatibility:
1. Updated nodes can't accept particles from the dashboard
    a. In that case, `par` will save our souls and will simply ignore any failures
2. Dashboard can't accept particles from the updated nodes
    a. In that case, the dashboard wouldn't be able to process particles and won't call callbacks, so from a user's standpoint nothing will break

Also, once we switch back to discovery via `findAndAskNeighboursSchema`, we will inevitably go through different nodes and wouldn't be able to control anything. The dashboard is meant to discover all services in all nodes on the network. If that causes bugs or breakage, that's good!